### ISSUE TITLE
How to work with Reactive Form?

### ISSUE THREAD

**Issue by sonphnt** [mariohmol/ang-jsoneditor#45]
Hi, Can we make this work with Reactive Form? I need to use with "formControlName"?

Thanks

**Comment by mariohmol** [comment]
Hi.. i just finished this integration with forms.. check v1.7.8

**Comment by gentlemanjohn** [comment]
I'm still having an issue with this on v1.8.2

```
<form [formGroup]="newAnswerForm" (submit)="addAnswer()">
  ...
      <json-editor formControlName="answer" [options]="options"></json-editor>
  ...
</form>
```
```
this.newAnswerForm = this.fb.group({
      name: [''],
      answer: [{
        "answer": "This is a string."
      }],
      exactMatch: [true]
    });
```
"This is a string" populates the editor on load, but when I make changes within the JSON editor they are not saved when I go to submit the form.  I also get an error in core.js:

`ERROR TypeError: Cannot read property 'session' of null`

**Comment by samudiogo** [comment]
Hi @mariohmol !
I forked you project and tried to fix this issue, no success. But perhaps his bug fires on jsoneditor:

https://github.com/josdejong/jsoneditor/blob/a78adf667634f570d2a4a41af2dc3557b957bd31/src/js/textmode.js#L721


    if (this.aceEditor) {
          // prevent emitting onChange events while setting new text
          this.onChangeDisabled = true;
          this.aceEditor.setValue(text, -1);
          if (clearHistory) {
              // prevent initial undo action clearing the initial contents
              var me_1 = this;
              setTimeout(function () {
                  me_1.aceEditor.session.getUndoManager().reset();
              }, 0);
          }
          this.onChangeDisabled = false;
      }


StackBlitz:
https://stackblitz.com/edit/angular-gx54gz

**Comment by samudiogo** [comment]
@mariohmol Bro, I reported this bug at https://github.com/josdejong/jsoneditor/issues/781
Using the version 7.0.2 is working fine!

**Comment by mariohmol** [comment]
thank you @samudiogo 

released a version 1.8.3 with the updated dependency for ang-jsoneditor

@gentlemanjohn could u please test again?

**Comment by mariohmol** [comment]
related with https://github.com/mariohmol/ang-jsoneditor/issues/51
